I've tried to test this on my rig as I have the same requirement and see the following output:

```
[2024-01-17 12:44:47,802] [INFO] [led] plugin loaded for /sys/class/leds/led0/brightness
Traceback (most recent call last):
  File "/usr/local/bin/pwnagotchi", line 154, in <module>
    display = Display(config=config, state={'name': '%s>' % pwnagotchi.name()})
[2024-01-17 12:44:47,805] [DEBUG] [led] event 'loaded' set
  File "/usr/local/lib/python3.7/dist-packages/pwnagotchi/ui/display.py", line 12, in __init__
    super(Display, self).__init__(config, hw.display_for(config), state)
  File "/usr/local/lib/python3.7/dist-packages/pwnagotchi/ui/hw/__init__.py", line 74, in display_for
[2024-01-17 12:44:47,808] [DEBUG] [led] using pattern 'oo  oo  oo oo  oo  oo  oo' ...
    return Waveshare213g(config)
NameError: name 'Waveshare213g' is not defined
```

Can be resolved by adding the import to pwnagotchi/ui/hw/__init__.py

--- a/pwnagotchi/ui/hw/__init__.py
+++ b/pwnagotchi/ui/hw/__init__.py
@@ -14,6 +14,7 @@ from pwnagotchi.ui.hw.waveshare29inch import Waveshare29inch
 from pwnagotchi.ui.hw.waveshare144lcd import Waveshare144lcd
 from pwnagotchi.ui.hw.waveshare154inch import Waveshare154inch
 from pwnagotchi.ui.hw.waveshare213d import Waveshare213d
+from pwnagotchi.ui.hw.waveshare213g import Waveshare213g
 from pwnagotchi.ui.hw.waveshare213bc import Waveshare213bc
 from pwnagotchi.ui.hw.waveshare213inb_v4 import Waveshare213bV4
 from pwnagotchi.ui.hw.waveshare35lcd import Waveshare35lcd

--- a/pwnagotchi/ui/hw/__init__.py
+++ b/pwnagotchi/ui/hw/__init__.py
@@ -14,7 +14,6 @@ from pwnagotchi.ui.hw.waveshare29inch import Waveshare29inch
 from pwnagotchi.ui.hw.waveshare144lcd import Waveshare144lcd
 from pwnagotchi.ui.hw.waveshare154inch import Waveshare154inch
 from pwnagotchi.ui.hw.waveshare213d import Waveshare213d
-from pwnagotchi.ui.hw.waveshare213g import Waveshare213g
 from pwnagotchi.ui.hw.waveshare213bc import Waveshare213bc
 from pwnagotchi.ui.hw.waveshare213inb_v4 import Waveshare213bV4
 from pwnagotchi.ui.hw.waveshare35lcd import Waveshare35lcd

